services/web/server/tests/unit/with_dbs/03/test_project_db.py

```diff
@@ -187,6 +192,21 @@ async def _inserter(prj: dict[str, Any], **overrides) -> dict[str, Any]:
         }
         default_config.update(**overrides)
         new_project = await db_api.insert_project(**default_config)
+        if default_config["project"].get("access_rights") or default_config[
+            "project"
+        ].get("accessRights"):
+            _access_rights = default_config["project"].get(
+                "access_rights", {}
+            ) | default_config["project"].get("accessRights", {})
```

Comment:
something like this should work as well
```suggestion
        if  _access_rights := default_config["project"].get(
                "access_rights", {}
            ) | default_config["project"].get("accessRights", {}):
```

Reply:
thanks